README.md

```diff
@@ -526,14 +526,44 @@ enum ImageOrientation {
 
 | Param | Type |
 | --- | --- |
-| environment | Environment | 
-| [venueApiUrl] | string | 
+| environment | Environment |
+| [venueApiUrl] | string |
 
 * getSeatAttributes(venueId) ⇒ Array&lt;SeatAttributes&gt;
+* getSeatAttributesBySeatIds(venueId, seatIdCollection) ⇒ Array&lt;SeatAttributes&gt;
+* getDetails(venueId) ⇒ Object&lt;VenueDetails&gt;
+* getChartDetails(productId, date) ⇒ Object&lt;ChartDetails&gt;
 
 #### ➥ getSeatAttributes(venueId) ⇒ Array&lt;SeatAttributes&gt;
 *Get list of seat attributes*
-**Returns**: Array&lt;SeatAttributes&gt; - *seat attribute models* 
+**Returns**: Array&lt;SeatAttributes&gt; - *seat attribute models*
+
+| Param | Type |
+| --- | --- |
+| venueId | string |
+| performanceDate | string |
```

Comment:
const getSeatAttributes = async (venueId: string, performanceDate?: string, performanceTime?: string) =>

Non-mandatory fields should be marked here (with square brackets)

Reply:
done